lesson 12/styles/style.css

```diff
@@ -293,14 +344,451 @@
+    display: inline-block;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #F3F3F4;
+    padding: 10px 17px 11px 16px;
+}
+
+.polygon {
+    fill: #383838;
+    width: 7px;
+    height: 19px;
+}
+
+.homes_text {
+    font-family: 'Roboto', sans-serif;
+    color: #BFBFBF;
+    font-size: 24px;
+    font-weight: 400;
+}
+
+.destinations_container {
+    width: 100%;
+    padding: 120px 104px 120px;
+    position: relative;
+    }
+
+.destinations_title {
+    font-family: 'Roboto', sans-serif;
+    text-decoration-color: #383838;
+    font-size: 40px;
+    font-weight: 500;
+    text-align: center;
+    margin-bottom: 105px;
+}
+
+.destinations__navi {
+    margin-bottom: 65px;
+}
+
+.destinations__list {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+.dest__a {
+    text-decoration: none;
+}
+
+.dest__list {
+    display: inline-block;
+    font-family: 'Roboto', sans-serif;
+    font-size: 24px;
+    text-align: center;
+    color: #383838;
+    background-color: #F3F3F4;
+    margin: 0 -3px;
+    padding: 17px 0;
+
+}
+
+.dest__regions {
+    margin-left: 0;
+    width: 192px;
+    height: 64px;
+    border-bottom-left-radius: 8px;
+    border-top-left-radius: 8px;
+}
+
+.dest__regions:hover {
+    border-radius: 8px;
+    background-color: var(--blue);
+    color: var(--white);
+}
+
+.dest__cities {
+    width: 192px;
+    height: 64px;
+}
+
+.dest__cities:hover {
+    border-radius: 8px;
+    background-color: #3077C6;
+    color: #FFFFFF;
+}
+
+.dest__places {
+    width: 240px;
+    height: 64px;
+    border-bottom-right-radius: 8px;
+    border-top-right-radius: 8px;
+}
+
+.dest__places-md {
+    display: none;
+}
+
+.dest__places:hover {
+    border-radius: 8px;
+    background-color: #3077C6;
+    color: #FFFFFF;
+}
+
+.destinations__block {
+    display: grid;
+    column-gap: 16px;
+    margin-bottom: 48px;
+}
+
+.destinations__img {
+    width: 100%;
+    max-width: 296px;
+    aspect-ratio: 1/1;
+    margin-bottom: 25px;
+}
+
+.dest__text {
+    display: block;
+    font-family: 'Roboto', sans-serif;
+    font-size: 24px;
+    color: #3077C6;
+}
+
+.maldives {
+    position: relative;
+}
+
+.book_now {
+    opacity: 0.2;
+    position: absolute;
+    z-index: 1;
+    top: 40%;
+    transform: translate(50%);
+    right: 50%;
+    width: 128px;
+    height: 48px;
+    border: 1px solid #3077C6;
+    border-radius: 8px;
+    background-color: #FFFFFF;
+    color: #3077C6;
+    font-family: 'Roboto', sans-serif;
+    font-size: 18px;
+    text-align: center;
+    padding: 13px 0;
+}
+
+.book_now:hover, .book_now:focus {
+    opacity: 1;
+    box-shadow: 0 4px 70px #383838;
+}
+
+.destinations__pointer {
+    display: inline-block;
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #F3F3F4;
+    padding: 8px 16px;
+    transform: rotate(90deg);
+    margin-left: 48%;
+    margin-right: 49%;
+}
+
+.sign__container {
```

Comment:
Please, check design. You can see that sign up block change position on mobile screen